verification/curator-service/api/src/geocoding/mapbox.ts

```diff
@@ -56,20 +67,40 @@ export default class MapboxGeocoder {
         });
     }
 
-    async geocode(query: string): Promise<GeocodeResult[]> {
+    async geocode(
+        query: string,
+        opts?: GeocodeOptions,
+    ): Promise<GeocodeResult[]> {
         query = query.trim();
```

Comment:
We're calling this with `location?.query` in the other file. Won't this line throw if it's undefined?

Reply:
good call on that bad call, I am now checking for the presence of query in the cases controller before calling this.